### ISSUE TITLE
[FR] CocoaPods Support

### ISSUE THREAD

**Issue by moetanahy** [Mordil/swift-currency#38]
Hi,

Love the library and excited to start using it.  However, I'm unable to install it - I'm not sure if this is the right place to ask.

I am using Cocoapods for my dependencies and am trying to figure out a way to install swift-currency using pods.  However, I sense it's not supported on this library (could be wrong!)

I've tried the following with no luck:

```
pod 'swift-currency', :git => 'https://github.com/peek-travel/swift-currency', :tag => '0.6.0'
```

Is there anything I need to do to get this working?  Is there another solution to get this working with Cocoapods?

Thanks

**Comment by Mordil** [comment]
Hi! This is definitely the place to ask - right now we don't support CocoaPods, only Swift Package Manager.

It's something we should support, I'll update this issue to make it a feature request

**Comment by moetanahy** [comment]
Thanks Mordil (and thanks for getting back to me so quickly!).  That would be excellent if it's supported as would love to use the library but don't want to change from cocoapods at this point.

**Comment by Mordil** [comment]
This should be available as of `0.6.1`, give it a try and let me know if you run into issues!

**Comment by moetanahy** [comment]
That's awesome - thanks Mordil! Thanks for getting this done so quickly!
